File: Java/Program/Product.java

class Product {
    private String idProduct;
    private String name;
    private String brand;
    private int price;

    public Product(){

    }

    public Product(String idproduct, String name, String brand, int price){
        this.idProduct = idproduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    public String getIdProduct() {
        return idProduct;
    }

    public void setIdProduct(String idProduct) {
        this.idProduct = idProduct;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    
}

File: Python/Program/Product.py
class Product:
    __id = ''
    __nama = ""
    __brand = ""
    __price = ""
    
    def __init__(self,id = '',nama = "",brand = "",price = ""):
        self.__id = id
        self.__nama = nama
        self.__brand = brand
        self.__price = price
    
        # Getter
    def get_id(self):
        return self.__id

    def get_nama(self):
        return self.__nama

    def get_brand(self):
        return self.__brand

    def get_price(self):
        return self.__price

    # Setter
    def set_id(self, id):
        self.__id = id

    def set_nama(self, nama):
        self.__nama = nama

    def set_brand(self, brand):
        self.__brand = brand

    def set_price(self, price):
        self.__price = price